doc/classes/PackedInt32Array.xml

```diff
@@ -60,6 +60,15 @@
 				Creates a copy of the array, and returns it.
 			</description>
 		</method>
+		<method name="fill">
+			<return type="void">
+			</return>
+			<argument index="0" name="value" type="int">
+			</argument>
+			<description>
+				Inserts a new integer at every position in the array.
```

Comment:
```suggestion
				Inserts a new element at every position in the array.
```

Other array types say "Inserts a new element", but integer arrays are specific about type.

I suggest either making the wording more general-purpose, or modify other method descriptions in a similar fashion as in this one. Making it more general-purpose could potentially ease translation, perhaps.